src/resources/UsageAnalytics/Read/Exports/ExportsInterfaces.ts

```diff
@@ -87,10 +87,88 @@ export interface GenerateExportParams extends EstimateExportParams {
     useDisplayNames?: boolean;
 }
 
+interface BaseGenerateExportWithBodyParams {
+    /**
+     * The beginning date of the date range. ISO8601 format 'YYYY-MM-DDThh:mm:ss.sssZ'
+     */
+    from: string;
+    /**
+     * The end date of the date range. ISO8601 format 'YYYY-MM-DDThh:mm:ss.sssZ'
+     */
+    to: string;
+    /**
+     * A file name for the export. Can only contain US-ASCII characters. If none is supplied it will be set to the export's id.
+     */
+    filename?: string;
+    /**
+     * Optional description for the export.
+     */
+    description?: string;
+    /**
+     * The dimensions that will be exported. If not provided, all dimensions will be exported.
+     */
+    dimensions?: string[];
+    /**
+     * The format of the generated CSV files. If not provided, it will default to Excel.
+     */
+    exportFormat?: CSVFileFormat;
+    /**
+     * Whether to use the display names in the export's header. If false, the api names will be used. If not provided, it defaults to false.
+     */
+    useDisplayNames?: boolean;
+}
+
+export interface GenerateExportWithBodyParams extends BaseGenerateExportWithBodyParams {
+    /**
+     * The filter that will be applied to the events common dimensions (shared by all types of events).
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    commonFilters?: string[];
+    /**
+     * The filter that will be applied to the click and search events dimensions.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    searchesFilters?: string[];
+    /**
+     * The filter that will be applied to the custom events dimensions.
+     * Multiple filter parameters are joined with the AND operator.
+     */
+    customEventsFilters?: string[];
+    /**
+     * The tables that will be exported. If not provided, all tables will be exported.
```

Comment:
```suggestion
     * The types of tables to export. By default, all types of tables are exported.
```

Reply:
The enum name `ExportTablesType` shouldn't have `Type` at the end (these are actual tables, not types of tables). I updated the previous enum name to indicate this.